pkg/reconciler/inmemorychannel/dispatcher/inmemorychannel_test.go

```diff
@@ -24,14 +24,16 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 
 	"github.com/google/go-cmp/cmp/cmpopts"
-	"knative.dev/eventing/pkg/kncloudevents"
 
 	"github.com/google/go-cmp/cmp"
 	"knative.dev/pkg/ptr"
 	"knative.dev/pkg/reconciler"
 
+	"knative.dev/eventing/pkg/kncloudevents"
+
 	corev1 "k8s.io/api/core/v1"
 	clientgotesting "k8s.io/client-go/testing"
+
```

Comment:
nits: seems that gofmt does do a good job here, should we cluster all the `knative.dev/` packages?